index.php

```diff
@@ -453,11 +453,12 @@
                         <img src="images/icons/categories/64/preferences-system-parental-controls.svg" alt="Icon of an adult holding hand the hand of a child"/>
                         Parental Controls
                     </h2>
-                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h4>Time Limits</h4>
+                    <h4>Implement parental controls or manage your own digital wellbeing.</h4>
+                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h3>Screen Time</h3>
```

Comment:
We don't actually say "Time Limits" in the UI, so I'm leaving this terminology change :)

```suggestion
                    <?php include('images/icons/actions/symbolic/appointment-symbolic.svg'); ?><h4>Screen Time</h4>
```